cypress/integration/login.js

```diff
@@ -1,27 +1,108 @@
+import LoginPage from '../objects/loginPage'
+import Timelog from '../objects/timelog'
+
+const loginPage = new LoginPage();
+const timelog = new Timelog();
 describe('Sourcebooks login', function() {
 
     it('Should display validation for empty user after attempted loggin', function () {
         
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
+        loginPage.visit('/');
+        loginPage.getvalidationerror().should('not.visible')

```

Comment:
according to camel case naming convention every new word in variable name should start with first capital letter
getvalidation**E**rror